timings/timings.go

```diff
@@ -46,6 +46,16 @@ var timeNow = func() time.Time {
 //   nestedTiming.Stop()
 //   t1.Stop()
 //   troot.Save()
+//
+// In addition, a few helpers exist to simplify typical use cases, for example the above example
+// can be reduced to:
+//   troot := timings.NewForTask(task) // tags set automatically, label and summary derived from task
+//   t1 := troot.StartSpan("computation", "...")
+//   t1.Run("sub-computation", "...", func(nested *Span) {
```

Comment:
timing.Run(t1...

Reply:
Thanks for catching all the inconsistencies in the comments, appreciated!